PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1816,6 +1816,35 @@ bool Notepad_plus::findInFilelist(std::vector<generic_string> & fileNames)
 
 	const bool isEntireDoc = true;
 
+	size_t MaxLineNum = 0;

```

Comment:
The variable name starts always with lowercase. Please check contribution guide line for the coding convention of this project.